documentation-site/examples/spinner/next-custom.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import * as React from 'react';
+import {StyledSpinnerNext} from 'baseui/spinner';
+
+export default () => (
+  <StyledSpinnerNext
+    $style={({$theme}) => ({
```

Comment:
We should use `withStyle` in the example since `$style` is not a recommended way for adding style overrides.

Reply:
pr to edit this: https://github.com/uber/baseweb/pull/2543